tests/Feature/AuthControllerLoginFailTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Http\Response;
+use Tests\TestCase;
+
+class AuthControllerLoginFailTest extends TestCase
+{
+    use RefreshDatabase;
+    use WithFaker;
+
+    public function testLogin()
+    {
+        // Create a user for testing
+        $user = User::factory()->create([
+            'email' => 'test@example.com',
+            'password' => bcrypt('password123'),
+        ]);
+
+
+        // Prepare login data
+        $data = [
+            'email' => 'test@example.com',
```

Comment:
'email' => $user->email;

also, you can add a password constant and use it in lines 21 and 28: